src/services/blockExplorer.js

```diff
@@ -86,34 +81,37 @@ function checkETHTransaction(data: Object): boolean {
     type.isString(data.gas) &&
     type.isString(data.from) &&
     type.isString(data.hash) &&
+    type.isString(data.input) &&
+    type.isString(data.nonce) &&
```

Comment:
As I remember, that getNonce method returns number. Please check that this is always a string

Reply:
I checked, etherscan API returns all fields as strings